### DISCUSSION TITLE
Can I use ydata-profiling to profile data in chunks?

### DISCUSSION THREAD

**Discussion by grojas123** [Data-Centric-AI-Community/fg-data-profiling#1546] [Q&A]
I am preventing an issue if I have too much data. Can I use ydata-profiling to profile data in chunks? 
I have data I will read on a database or CSV and I want to process this data on Chunks, I am thinking of the inconvenience of reading all in memory at once. If the query/CSV will be so big .
What are my options here?

**Comment by OdyAsh** [comment]
up
